Sub-app-module tky and Wallet Module Tested

macuare opened the pull request:
Several Packaged tested.
There are a lot of variable coming from interfaces can not be tested beacuse they belongs to the fermat core. Internet connection is needed.

darkestpriest commented:
Try to indicate the issue status like that:
* #IssueNumber 30%
or
* #IssueNumber add some methods, to implement: new method

macuare committed:
Test: Se agregaron pruebas al modulo, modificaciones al gradle. Aun quedan por evaluar algunas excepciones e interfaces. Existen variables que estan declaradas pero de forma de package....esto dificulta conocer si realmente era de acceso publico o privado. Hay otras variables que dificilmente se pueden manipular porque pertenecen al core.

macuare committed:
Test: testing modules ,

Gradle modifications were made. There are still some exceptions to evaluate and interfaces. There are variables declared package therefore not known whether they are public or private access . There are other variables are difficult to handle, because the core fermat belong .

macuare committed:
Test: testing modules ,

Gradle modifications were made. There are still some exceptions to evaluate and interfaces. There are variables declared package therefore not known whether they are public or private access . There are other variables are difficult to handle, because the core fermat belong .

macuare committed:
Test: testing modules ,

Gradle modifications were made. There are still some exceptions to evaluate and interfaces. There are variables declared package therefore not known whether they are public or private access . There are other variables are difficult to handle, because the core fermat belong .

macuare committed:
Test: testing modules ,

Gradle modifications were made. There are still some exceptions to evaluate and interfaces. There are variables declared package therefore not known whether they are public or private access . There are other variables are difficult to handle, because the core fermat belong .

macuare committed:
Test: testing modules ,

Gradle modifications were made.  There are variables declared package therefore not known whether they are public or private access . There are other variables are difficult to handle, because the core fermat belong .
New Package tested.

macuare committed:
Test: testing modules ,

Several Packaged tested.
There are a lot of variable coming from interfaces can not be tested beacuse they belongs to the fermat core. Internet connection is needed.
